GenumKit/Stencil/Filters.swift

```diff
@@ -106,3 +126,83 @@ struct NumFilters {
     return "\(percent)%"
   }
 }
+
+struct ModelParsingFilters {
```

Comment:
The main concern here is that this will grow as people add their own templates. This is unfortunate, but sadly, Stencil just really doesn't support lists well because the lack of a `map`  (oh, how I love `map`). As people more methods details, we should be able to change these up to be as generic as we need. I tried to keep stuff as generic as possible creating lists for joining and not just outputting strings.